src/Uno.UI.RuntimeTests.Engine.Library/UnitTestsControl.cs

```diff
@@ -271,9 +276,12 @@ void Setter()
             {
                 _applicationView.Title = message;
             }
+
+            tcs.SetResult(default!);
         }
 
-        await Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, Setter);
+        RunOnDispatcher(Setter);
+        await tcs.Task;
```

Comment:
there is no awaitable equivalent for DispatcherQueue.
we could inline the `tcs` implementation to RunOnDispatcher and return an awaitable `Task`.
that would greatly simplify the code, at the cost of double wrapping with delegate.

Reply:
Yeah, that's probably best to have RunOnDispatcher be awaitable, double wrapping is not a problem here.